frc53_nft/src/lib.rs

```diff
@@ -412,6 +415,56 @@ where
         Ok(self.state.transfer_return(&self.runtime, intermediate)?)
     }
 
+    /// Enumerates a page of TokenIDs
+    pub fn list_tokens(&self, cursor: Option<AmtCursor>, max: u64) -> Result<ListTokensReturn> {
+        let (tokens, next_cursor) = self.state.list_tokens(&self.runtime, cursor, Some(max))?;
+        Ok(ListTokensReturn { tokens, next_cursor })
+    }
+
+    /// Enumerates a page of TokenIDs owned by a specific address
+    pub fn list_owned_tokens(
+        &self,
+        owner: &Address,
+        cursor: Option<AmtCursor>,
+        max: u64,
+    ) -> Result<ListTokensReturn> {
+        let owner_id = self.runtime.resolve_id(owner)?;
+        let (tokens, next_cursor) =
+            self.state.list_owned_tokens(&self.runtime, owner_id, cursor, Some(max))?;
+        Ok(ListTokensReturn { tokens, next_cursor })
+    }
+
+    /// Returns all the operators approved by an owner for a token
+    pub fn list_token_operators(&self, owner: &Address, token_id: TokenID) -> Result<Vec<ActorID>> {
+        let _owner_id = self.runtime.resolve_id(owner)?;
+        Ok(self.state.list_token_operators(&self.runtime, token_id)?)
+    }
+
+    /// Enumerates tokens for which an account is an operator for an owner
+    pub fn list_operator_tokens(
+        &self,
+        owner: &Address,
```

Comment:
Does the same apply here, that we should just iterate tokens for which an account is an operator, without specifying an owner. Otherwise how would the caller figure out which owners are worth passing here anyway?

Reply:
Yeh, makes sense to remove `owner` param here